src/routes/tokenGovernance.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Tabs } from 'antd';
+
+import Web3SubmitWarning from '../components/web3SubmitWarning';
+
+import { Button, Form, Input, Row, Col } from 'antd';
+import TextArea from 'antd/lib/input/TextArea';
+import { keccak256 } from 'ethereumjs-util';
+import Web3 from '../stores/web3/ts_workaround';
+import { tokenGovernanceContractStore } from '../stores/tokenGovernanceContract';
+
+const { Fragment } = React;
+const { TabPane } = Tabs;
+
+export default class TokenGovernance extends React.Component {
+  public state: { title: string; description: string };
+  public sendProposal: any;
+
+  constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      title: '',
+      description: '',
+    };
+  }
+
+  public handleClick = e => {
+    this.setState({
```

Comment:
you don't need to call `setState` as we are using MobX. With MobX you can mark state as "observable" (e.g. using @observable decorator) and then just change it normally. [Quick intro to MobX](https://mobx.js.org/README.html)

Reply:
i will read through that and change it later on. it dosent work for me yet when i add [@obersvable](url) instead of `this.state`